lib/zendesk_api/client.rb

```diff
@@ -143,8 +143,6 @@ def self.check_deprecated_namespace_usage(attributes, name)
     # Retry middleware if retry is true
     def build_connection
       Faraday.new(config.options) do |builder|
-        builder.request :multipart
```

Comment:
Why do we remove "multipart"?

Reply:
it's already present down the line, before the Upload middleware. this was a duplicate.